### PR TITLE
Lower op embedding bag

### PR DESCRIPTION
**Pull request by bhavya01**

### CODE REVIEW

File: infra/tpu-pytorch/infra_triggers.tf [hunk 1]
```diff
@@ -7,4 +7,4 @@ module "terraform_apply" {
 
   worker_pool_id = module.worker_pool.id
   location       = "global"
-}
+}
```
Comment (will-cromar) [inline infra/tpu-pytorch/infra_triggers.tf:10]: Is this change intentional?

Comment (wonjoolee95) [reply infra/tpu-pytorch/infra_triggers.tf:10]: Seems like Manfei's commit accidentally got into the PR. @bhavya01 can you fix the rebase on this PR?

**Comment by bhavya01** [comment]
@wonjoolee95 This should be ready for review. I implemented the _embedding_bag_forward_only op for now. It seems that a lot of tests fail without the backward pass implementation.

**Comment by wonjoolee95** [comment]
Thanks! I'll take a look at this today.

**Comment by bhavya01** [comment]
@wonjoolee95 Do you think that you have time to look at this PR?

**Comment by wonjoolee95** [comment]
Apologies for missing this PR. Changes LGTM, thanks!

I noticed an unrelated commit `[Update infra_triggers.tf](https://github.com/pytorch/xla/pull/6574/commits/6dccf0a02d7828516bdb589f2ae0dc79b64488fa)`, can you fix the rebase?
